vsock_proxy/Cargo.toml

```diff
@@ -12,6 +12,7 @@ chrono = "0.4"
 clap = "3.2"
 dns-lookup = "2.0.3"
```

Comment:
Looks like `dns-lookup` is not used anywhere anymore. We should probably remove it here as well.